pensjon-brevbaker/src/main/kotlin/no/nav/pensjon/etterlatte/maler/fraser/barnepensjon/BarnepensjonFellesFraser.kt

```diff
@@ -186,51 +192,122 @@ object BarnepensjonFellesFraser {
+                        Nynorsk to "Barnepensjon er skattepliktig, men ikkje trekkpliktig.",
+                        English to "Child pension is taxable, but not subject to withholding tax.",
+                    )
                 }
             } orShow {
                 paragraph {
                     text(
-                        Bokmal to "Barnepensjon er skattepliktig, men ikke trekkpliktig. Du kan lese mer om skattetrekk i vedlegget “Informasjon til deg som mottar barnepensjon”.",
-                        Nynorsk to "Barnepensjon er skattepliktig, men ikkje trekkpliktig. Du kan lese meir om skattetrekk i vedlegget “Informasjon til deg som får barnepensjon”.",
-                        English to "Child pension is taxable, but not subject to withholding tax. You can read more about tax deductions in the attachment “Information to recipients of children’s pensions”.",
+                        Bokmal to "Barnepensjon er skattepliktig, men ikke trekkpliktig.",
+                        Nynorsk to "Barnepensjon er skattepliktig, men ikkje trekkpliktig.",
+                        English to "Child pension is taxable, but not subject to withholding tax.",
                     )
                 }
             }
-
-            showIf(etterbetaling) {
+            showIf(etterbetaling.notNull()) {
                 paragraph {
                     text(
-                        Bokmal to "Du får etterbetalt pensjon. Vanligvis vil du få denne i løpet av " +
-                                "tre uker. Hvis Skatteetaten eller andre ordninger har krav i etterbetalingen kan " +
-                                "denne bli forsinket. Fradrag i etterbetalingen vil gå fram av utbetalingsmeldingen.",
-                        Nynorsk to "Du får etterbetalt pensjon. Normalt sett får du denne i løpet av " +
-                                "tre veker. Dersom Skatteetaten eller andre ordningar har krav i etterbetalinga, kan " +
-                                "denne bli forseinka. Frådrag i etterbetalinga vil gå fram av utbetalingsmeldinga.",
-                        English to "You will receive a back payment on your pension. You will usually " +
-                                "receive this back payment within three weeks. If the Norwegian Tax Administration or " +
-                                "other schemes are entitled to the back payment, the payment to you may be delayed. " +
+                        Bokmal to "Du får etterbetalt pensjon. Vanligvis vil du få denne i løpet av tre uker. ",
+                        Nynorsk to "Du får etterbetalt pensjon. Vanlegvis vil du få denne i løpet av tre veker. ",
+                        English to
+                            "You will receive a back payment on your pension. " +
+                            "You will usually receive this back payment within three weeks. ",
+                    )
+                    showIf(etterbetaling.inneholderKrav_safe.equalTo(true)) {
```

Comment:
Hva bruker `_safe` til? Ikke sett den brukt før

Reply:
For å nå variablene under `etterbetaling`